examples/dag-with-args-example/README.md

```diff
@@ -45,7 +45,7 @@ To run this example, you need:
 * Versatile Data Kit
 * Trino DB
 * `vdk-trino` - VDK plugin for a connection to a Trino database
-* [VDK DAGs](https://github.com/vmware/versatile-data-kit/tree/main/specs/vep-1243-vdk-meta-jobs)
+* [VDK DAGs](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins/vdk-meta-jobs)
```

Comment:
nit: may be valuable to have "here is the README, here is the specification" to have the level of technical debt covered based on audience preferences

Reply:
Although I find the README entirely user-oriented, I am OK with that. Makes sense. Changed it.